core/domain/exp_services_test.py

```diff
@@ -1673,6 +1682,17 @@ class ExplorationCommitLogUnitTests(ExplorationServicesUnitTests):
         'post_commit_status': 'public'
     }
 
+    COMMIT_UPDATE_FIRST_PUBLISHED_MSEC_EXP_2 = {
```

Comment:
Hm, the way to merge the 2 commits would be to add the first_published change in the _change_activity_status() function, and _change_activity_status is called under _publish_activity. Do you think it would be intuitive to merge the two?